lib/adapter.js

```diff
@@ -155,6 +159,7 @@ class MochaAdapter {
         if (err) {
             this.runner.lastError = err
         }
+        process.emit(event, message)
```

Comment:
@georgecrawford is [this](https://github.com/webdriverio/webdriverio/pull/896/files#diff-b3b53682a18f203ac8d29b0e277cad26R105) the reason why you need to emit the payload?